experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -233,11 +233,14 @@ export const makeEditGenerator = (
 				// It'd be reasonable to move this to config. The idea is that by avoiding large deletions,
 				// we're more likely to generate more interesting outcomes.
 				const count = state.random.integer(1, Math.min(3, field.length - start));
+				const node = field.at(start);
```

Comment:
Perhaps use 'atIndex()' here?

Reply:
Would need to cast `field` which is just `Sequence` (not `LazySequence`) and thus doesn't have it.